vendor/src/github.com/bsphere/le_go/certs.go

```diff
@@ -0,0 +1,222 @@
+package le_go
+
+// Certs from https://logentries.com/doc/certificates
+var pemCerts = []byte(`
```

Comment:
@cpuguy83 makes sense... I guess I could try to open a PR to le_go to fix that...